docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Berlin
+
+RUN apt update \
+    && apt-get install -y \
+    g++ \
+    cmake \
+    libatlas-base-dev \
+    python3 \
+    python3-dev \
+    python3-pip \
+    swig \
+    git\
+    libhdf5-serial-dev\
+    && ln -sf /usr/bin/swig4.0 /usr/bin/swig
+    
+RUN pip3 install python-libsbml>=5.17.0
+
+COPY amici.tar.gz /tmp
+
+ENV AMICI_CXXFLAGS -fopenmp
+ENV AMICI_LDFLAGS -fopenmp
+
+RUN pip3 install -U --upgrade pip wheel \
+    && mkdir -p /tmp/amici/ \
+    && cd /tmp/amici \
+    && tar -xzf ../amici.tar.gz \
+    && cd /tmp/amici/python/sdist \
+    && python3 setup.py -v sdist \
+    && pip3 install -v $(ls -t /tmp/amici/python/sdist/dist/amici-*.tar.gz | head -1)[petab,pysb] \
+    && rm -rf /tmp && mkdir /tmp
+
+# RUN pip3 install git+https://github.com/ICB-DCM/pyPESTO.git@develop#egg=pypesto
+RUN pip3 install pyPESTO jupyter pyswarm dlib
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5461541)